packages/maskbook/src/social-network-provider/twitter.com/ui/injectSearchResultBox.tsx

```diff
@@ -0,0 +1,24 @@
+import { MutationObserverWatcher } from '@dimensiondev/holoflows-kit/es'
+import { SearchResultBox } from '../../../components/InjectedComponents/SearchResultBox'
+import { Flags } from '../../../utils/flags'
+import { renderInShadowRoot } from '../../../utils/shadow-root/renderInShadowRoot'
+import { searchResultHeadingSelector } from '../utils/selector'
+import { twitterUrl } from '../utils/url'
+
+export function injectSearchResultBoxAtTwitter() {
+    if (location.hostname.indexOf(twitterUrl.hostIdentifier) === -1) return
```

Comment:
How could it happen? If host name does not match, the corresponding UI definition won't be activated